packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/components/plate-ui/templates-toolbar-button.tsx

```diff
@@ -56,13 +61,12 @@ const EmbedButton: React.FC<EmbedButtonProps> = ({ editor, templates }) => {
             onChange={filterChange}
           />
         )}
-        <DropdownMenuSeparator />
         {filteredTemplates.map((template) => (
           <DropdownMenuItem
             key={template.name}
             onMouseDown={(e) => {
               e.preventDefault();
-              setOpen(false);
+              openState.onOpenChange(false);
```

Comment:
in line with my other suggestion
```suggestion
              onOpenChange(false);
```